app/src/main/java/mollie/tictactoe/players/MobilePlayer.java

```diff
@@ -0,0 +1,29 @@
+package mollie.tictactoe.players;
+
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.Marks;
+
+public class MobilePlayer implements Player {
+
+    private final Marks mMark;
```

Comment:
@nickbdyer thats a naming conflict that goes back to my jar, but I agree it's confusing...